release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -42,6 +42,25 @@ The scope of support for layered and dependent components of {product-title} cha
 
 This release adds improvements related to the following components and concepts.
 
+[id="ocp-4-10-documentation"]
+=== Documentation
+
+[id="ocp-4-10-getting-started"]
+==== Getting started with {product-title}
+
+{product-title} 4.10 now includes a getting started guide. Getting Started with {product-title} defines basic terminology and provides role-based next steps for developers and administrators.
+
+The tutorials walk new users through the web console and the OpenShift CLI (`oc`) interfaces. New users accomplish the following tasks through the Getting Started.
+
+* How to create a project
+* Deploy a container image
+* Add a Python application
+* Scale applications
+* Create routes and grant view permissions
+* Connect to a database, load, and view your application.
```

Comment:
`Connect to a database, load, and view your application.`
This seems wrong. This means  Connect to a database, connect to a load, and connect to a view your application. Not sure what the fix is. 
`Connect to a database to load and view your application.`
`Connect to a database, then load and view your application.`

Reply:
@mburke5678 My apologies - in a rush, I left out the word "load data". Does this sound better?

Connect to a database, load your data, and view your application.